dashboard/src/components/Header/Menu.scss

```diff
@@ -105,8 +105,14 @@
   margin-bottom: 0.5rem;
   text-align: end;
   font-size: smaller;
-  a:hover {
-    color: #fff;
+  a {
+    color: var(--cds-global-color-blue-400, #2ec0ff) !important;
+  }
+  &:visited {
+    color: var(--cds-global-color-lavender-300 #8b8dda) !important;
+  }
+  &:hover {
+    color: var(--cds-global-color-blue-500, #fff) !important;
   }
```

Comment:
why this changes? (we should avoid using !important if possible).

Reply:
The `a:link` in the clr-ui css is taking more priority than the css style defined at Menu.scss, so, if not adding the `!important` the bad color won't get overidden.
Pehaps I'm missing something here :S

![image](https://user-images.githubusercontent.com/11535726/108833700-7e248480-75cd-11eb-809b-0efceeec4307.png)

Reply:
you have the answer :) if you add the `:link` modifier to the line `108`, you don't need the `!important` rule

Reply:
Thanks, after some changes we can get rid of the `!important` modifiers :)